### PR TITLE
BZ#2041307 added a note to the unidling applications process

### PR DESCRIPTION
**Pull request by sheriff-rh**
Version(s):
4.6, 4.7

Issue:
https://bugzilla.redhat.com/show_bug.cgi?id=2041307

Link to docs preview (VPN required):
[Unidling applications](http://file.rdu.redhat.com/antaylor/06-09-22/bz2041307/applications/idling-applications.html#idle-unidling-applications_idling-applications)

Additional information:
@andreaskaris pointed out that in 4.6 and 4.7 that under certain circumstances, applications do not unidle correctly. The in-depth explanation is provided in a KCS linked in the note.

### CODE REVIEW

**Comment by sheriff-rh** [comment]
@zhaozhanqi  please review as well, thanks

**Comment by sheriff-rh** [comment]
Rebased and updated references in the latest push. @kalexand-rh , can you approve this non-main branch PR when you get a chance? This note addresses an issue present in 4.6 and 4.7 only, as it is resolved in 4.8+. SME and QE approved. I'll send it to peer review next. Thanks!

**Review by kalexand-rh — COMMENTED** [review]
You're good to use the bot to CP this.

File: modules/idle-unidling-applications.adoc [hunk 1]
```diff
@@ -30,3 +30,8 @@ HAProxy router.
 ====
 Services do not support automatic unidling if you configure Kuryr-Kubernetes as an SDN.
 ====
+
+[NOTE]
+====
+In {product-title} {product-version}, idled services do not become active again when they receive network traffic if they were idled while the leader `ovnkube-master` process was restarted or after the `ovnkube-master` leader changed. For more information, refer to link:https://access.redhat.com/solutions/6671241[Knowledgebase Article 6671241].
```
[the remark this answers is not in the document]
Comment (sheriff-rh) [reply modules/idle-unidling-applications.adoc:36]: Will do, thank you!

**Pull request merged**